pkg/service/controller/servingcert/controller.go

```diff
@@ -193,7 +193,8 @@ func (sc *ServiceServingCertController) syncService(key string) error {
 
 	dnsName := service.Name + "." + service.Namespace + ".svc"
 	fqDNSName := dnsName + "." + sc.dnsSuffix
-	servingCert, err := sc.ca.MakeServerCert(sets.NewString(dnsName, fqDNSName))
+	expirationDate := time.Now().Add(crypto.DefaultCertificateLifetime)
```

Comment:
I want these certs to have a lifespan default independent of what that package says.  A local constant would do.